editor/src/clj/editor/bundle_dialog.clj

```diff
@@ -721,7 +721,7 @@
                           install-app-check-box
                           launch-app-check-box]
     (prefs/set! prefs [:bundle :ios :sign] (ui/value sign-app-check-box))
-    (set-string-pref! prefs [:bundle :ios :code-signing-identity] (ui/value code-signing-identity-choice-box))
+    (set-string-pref! prefs [:bundle :ios :code-signing-identity] (or (ui/value code-signing-identity-choice-box) ""))
```

Comment:
`(ui/value code-signing-identity-choice-box)` could return `nil` if there were no code-signing identities found. In this case, we fall back to `""`. There are other choice boxes used in this file, but those all have a non-empty list of options and a default, so in the other cases it's impossible to get `nil`.